src/elements/content-sidebar/activity-feed/annotations/AnnotationActivity.js

```diff
@@ -91,6 +91,11 @@ const AnnotationActivity = ({
                         isDisabled={isFileVersionUnavailable}
                         message={activityLinkMessage}
                         onClick={handleOnSelect}
+                        resinTags={{
+                            'data-resin-iscurrent': isCurrentVersion,
+                            'data-resin-itemid': id,
+                            'data-resin-target': isCurrentVersion ? 'annotationPageLink' : 'annotationVersionLink',
```

Comment:
Is changing the target necessary if we are adding `data-resin-iscurrent`?

Reply:
Yeah I also noticed that the information is redundant here. But maybe it's faster for product to get the query result by reducing one query condition?

Reply:
Can we double-check with them? Would be great to simplify this if possible.

Reply:
Sure